Makefile

```diff
@@ -35,8 +35,8 @@ DOCKER_BUILDX_CMD ?= docker buildx
 IMAGE_BUILD_CMD ?= $(DOCKER_BUILDX_CMD) build
 IMAGE_BUILD_EXTRA_OPTS ?=
 STAGING_IMAGE_REGISTRY := us-central1-docker.pkg.dev/k8s-staging-images
-IMAGE_REGISTRY ?= $(STAGING_IMAGE_REGISTRY)/kueue
 IMAGE_NAME := kueue
+IMAGE_REGISTRY ?= $(STAGING_IMAGE_REGISTRY)/$(IMAGE_NAME)
```

Comment:
revert, actually, long term I would like to not use the IMAGE_NAME as variable at all. It is just complicating the readability. We never replace it, so it only pretends to be a varibale.